### PR TITLE
[prometheus-postgres-exporter] Fix the parameter to disable `stat_bgwriter` collecting

### PR DESCRIPTION
**Pull request by mjasion**
Postgres-exporter [0.13.0](https://github.com/prometheus-community/postgres_exporter/commit/db08eee988e8e59095d780b620d4c9b8e8cc8ba8#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) renamed the parameter from `bgwriter` to  `stat_bgwriter`

#### What this PR does / why we need it
Fix the parameter name

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by zeritti** [comment]
Thank you, @mjasion. Will you raise the chart version in `Chart.yaml`?

**Comment by mjasion** [comment]
@zeritti done :-)

**Comment by mjasion** [comment]
@zeritti I converted it to draft, I see other parameters got renamed too 🙈

**Comment by mjasion** [comment]
@zeritti I fixed also stat_database collector. Ready ✅

File: charts/prometheus-postgres-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -52,10 +52,10 @@ spec:
           - "--extend.query-path=/etc/config.yaml"
           - "--web.listen-address=:{{ .Values.service.targetPort }}"
           {{- if .Values.config.disableCollectorDatabase }}
-          - "--no-collector.database"
+          - "--no-collector.stat_database"
```
Comment (zeritti) [inline charts/prometheus-postgres-exporter/templates/deployment.yaml:55]:
```suggestion
          - "--no-collector.database"
```

Thank you, @mjasion. The [database collector](https://github.com/prometheus-community/postgres_exporter/blob/master/collector/pg_database.go) and the [stat_database collector](https://github.com/prometheus-community/postgres_exporter/blob/master/collector/pg_stat_database.go) are two different collectors, i.e. this is not a case of a changed flag. The chart currently supports disabling the former but not the latter.

Comment (mjasion) [reply charts/prometheus-postgres-exporter/templates/deployment.yaml]: Ah, didnt know. Thx for catching this.

**Review by zeritti — APPROVED** [review]

**Pull request merged**